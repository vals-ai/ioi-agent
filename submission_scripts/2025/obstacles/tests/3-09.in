version https://git-lfs.github.com/spec/v1
oid sha256:dd809650356d7164efe1b928b4c52319d41812300e0f0397206f17711eb21b13
size 4777992
