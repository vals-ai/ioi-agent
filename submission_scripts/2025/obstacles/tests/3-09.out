version https://git-lfs.github.com/spec/v1
oid sha256:30b675c607cb0da6a5ae84d60626deb38b34d0f4972bcccfd6d8eeeec5e159ad
size 400036
